proto/chorus/chorus.proto

```diff
@@ -9,10 +9,13 @@ import "google/protobuf/empty.proto";
 
 
 service Chorus {
+  // Get app version
+  rpc GetAppVersion(google.protobuf.Empty) returns (GetAppVersionResponse);
```

Comment:
add mapping to [http.yaml](https://github.com/clyso/chorus/blob/main/proto/http.yaml) for REST endpoint

Reply:
Added